clusters/build-clusters/01_cluster/machine_config/99-debug-ssh.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   config:
     ignition:
-      version: 2.2.0
```

Comment:
These files are used for the CI infrastructure itself.  I'm not sure they're actually used right now.

But also since these are `MachineConfig` objects, the MCO will handle converting them internally.  I think it's simplest to keep them as spec2x for now.

Reply:
Dropped from current PR.